### ISSUE TITLE
no-descending-specificity

### ISSUE THREAD

**Issue by in19farkt** [kristerkari/stylelint-scss#258]
Hello! I encountered such a problem:
`Expected selector "#{$r}_modifier .block__element" to come before selector ".block__element" (no-descending-specificity)`.
> What CSS is needed to reproduce this issue?
```sass
.block {
  $r: &;

  &__element {
    color: white;

    #{$r}_modifier & {
      color: black;
    }
  }
}
```
Any ideas?

**Comment by kristerkari** [comment]
Thanks @in19farkt. That rule is not actually part of this library, and I'm not familiar with the implementation details. Could you instead report the bug in stylelint's main repo? https://github.com/stylelint/stylelint

**Comment by in19farkt** [comment]
This problem occurs when a variable is inserted into the selector. Each preprocessor has its own syntax for this. In SCSS it looks like an `#{$variable}_modifier {}`. Maybe it's worth overriding this rule?

**Comment by kristerkari** [comment]
What we usually do with rules that come from `stylelint` library is that the bug should be reported in that repository first, and if it is not possible to fix in the original rule, then we can think of creating a new rule that is specific to the SCSS syntax.

Handling Sass' interpolation means a lot of complexity for the linting rules, so in your case it might make sense to use  `// stylelint-disable-line` for the selector that has the interpolation.

**Comment by in19farkt** [comment]
I submitted new [issue](https://github.com/stylelint/stylelint/issues/3455) to `stylelint`.

**Comment by kristerkari** [comment]
Thanks! It seems to already be fixed in stylelint, so I guess that nothing needs to be done here 👍
